src/components/RunPlanButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button } from "@mui/material";
+
+import { createAndStartTask, type TaskRequest } from "../utils/api";
+
+type RunPlanButtonProps = {
+  name: string;
+  params: object;
+  instrumentSession: string;
+};
+
+const RunPlanButton = ({
+  name,
+  params,
+  instrumentSession,
+}: RunPlanButtonProps) => {
+  return (
+    <Button
+      variant="contained"
+      sx={{ width: "150px" }}
+      onClick={async () => {
+        const taskRequest: TaskRequest = {
+          name: name,
+          params: params,
+          instrument_session: instrumentSession,
+        };
+        await createAndStartTask(taskRequest);
+      }}
+    >
+      Run Plan
```

Comment:
I'd have this say just Run (or Submit.. we'd have a queue eventually) but no plan - I'd try to avoid leaking bluesky terms as much as possible.